mkosi/distributions/opensuse.py

```diff
@@ -21,47 +21,131 @@ def filesystem(cls) -> str:
 
     @classmethod
     def install(cls, state: "MkosiState") -> None:
+        if state.config.base_image:
+            # We assume that the base image has been properly initialized and it
+            # contains all the metadata we need to install the additional
+            # packages.
+            return zypper_install(state, {*state.config.packages})
+
         return install_opensuse(state)
 
+    @classmethod
+    def remove_packages(cls, state: "MkosiState", packages: list[str]) -> None:
+        zypper_remove(state, packages)
+
     @staticmethod
     def initrd_path(kver: str) -> Path:
         return Path("boot") / f"initrd-{kver}"
 
 
+def invoke_zypper(state: MkosiState,
+                  global_opts: set[str],
+                  verb: str,
+                  verb_opts: set[str],
+                  *args: Any,
+                  with_apivfs: bool = False) -> None:
+    zypp_conf = state.root.joinpath("etc/zypp/zypp.conf")
+
+    cmdline = ["zypper", "--root", state.root, *global_opts, verb, *verb_opts, *args]
+
+    env=dict(ZYPP_CONF=zypp_conf)
```

Comment:
```suggestion
    cmdline = ["zypper", "--root", state.root, *global_opts, verb, *verb_opts, *args]
    env={"ZYPP_CONF": state.root.joinpath("etc/zypp/zypp.conf")}

```
since `zypp_conf` is not used anywhere else.